requirements.txt

```diff
@@ -93,8 +94,6 @@ MarkupSafe==1.1.1
 maxminddb==1.4.1
 meld3==2.0.0
 mixpanel==4.5.0
-mock==3.0.5
```

Comment:
Keep mock as it is used in a number of tests.

Reply:
It should be included in the standard lib